Makefile

```diff
@@ -52,11 +52,16 @@ tidy: ## Update dependencies
 clean: ## Remove binaries and test artifacts
 	@rm -rf bin
 
-generate: controller-gen ## Generate code and manifests
+generate: c-gen patch
+
+c-gen: controller-gen ## Generate code and manifests
 	$(Q)$(CONTROLLER_GEN) crd:crdVersions=v1 output:crd:dir=./manifests paths=./api/...
 	$(Q)$(CONTROLLER_GEN) schemapatch:manifests=./manifests output:dir=./manifests paths=./api/...
 	$(Q)$(CONTROLLER_GEN) object:headerFile=./hack/boilerplate.go.txt paths=./api/...
 
+patch: kustomize ## Apply kustomize patches
+	$(Q)$(KUSTOMIZE) build -o /manifests/core.rukpak.io_bundles.yaml ./manifests
```

Comment:
Perhaps instead of committing the final manifest output, we should add kustomization targets and use kustomize in a just-in-time mode? When we're actually applying manifests to the cluster? For example, `make install-crds` could call:
```
kubectl apply -k kustomize/crd
```
(assuming we put our CRD kustomization in the kustomize/crd directory)

This approach could also mean we don't need to carry the extra kustomize binary dependency (though it does raise questions about the version of kubectl that's used)

Otherwise, we'll have to make sure things stay in sync.